pkg/menu/menu.go

```diff
@@ -71,3 +72,29 @@ func GetInput(introwords string, location int, wid int, ht int, isValid func(str
 	uiEvents := ui.PollEvents()
 	return processInput(input, warning, isValid, uiEvents)
 }
+
+// create a new ui window and display an input box.
+func NewCustomInputWindow(introwords string, wid int, ht int, isValid func(string) (string, bool)) (string, error) {
```

Comment:
Also I think you're probably already planning to do this judging by the name of this function but just in case, would recommend also having a function for new input window that doesn't take in wid and ht and just uses fixed values since you tend to use the same values most of the time!

Reply:
yes I have one, but just haven't commit  it yet